pyrate/core/gamma.py

```diff
@@ -226,29 +227,45 @@ def parse_dem_header(path):
     return subset
 
 
-def parse_baseline_header(path):
-    """20060828-20061211_utm_cc.tif
-    Returns dictionary of Baseline metadata required for PyRate
+def parse_baseline_header(path: str) -> dict:
+    """
+    Returns dictionary of Baseline metadata required for PyRate.
+    Will read the Precise baseline estimate, if available,
+    otherwise will read the Initial baseline estimate.
 
-    :param str path: `Full path to Gamma *base.par file`
+    :param path: Full path to Gamma *base.par file
 
-    :return: subset: subset of full metadata
-    :rtype: dict
+    :return: bdict: Dictionary of baseline values
     """
-    lookup = _parse_header(path)
+    lookup = _parse_header(path)  # read file contents in to a dict
+
+    # split the initial and precise baselines
+    initial = lookup[GAMMA_INITIAL_BASELINE]
+    initial_rate = lookup[GAMMA_INITIAL_BASELINE_RATE]
+    precise = lookup[GAMMA_PRECISION_BASELINE]
+    precise_rate = lookup[GAMMA_PRECISION_BASELINE_RATE]
+
+    # read the precise baseline if all components are non-zero
+    # otherwise read the initial baseline
+    if float(precise[0]) == 0.0 and float(precise[1]) == 0.0 and float(precise[2]) == 0.0:
```

Comment:
Is there any situation where one of the baseline coordinates might be precisely zero, or does this always indicate that we should use the initial estimate instead?

Comment says _all_ components non-zero, code tests if _any_ components are non-zero.